### PR TITLE
fix run on MacOS

### PR DESCRIPTION
**Pull request by hellxcz**
linked issue : https://github.com/vlingo-net/vlingo-net-http/issues/5

### CODE REVIEW

File: src/Vlingo.Http.Tests/Resource/ResourceTestFixtures.cs [hunk 1]
```diff
@@ -37,7 +37,7 @@ public abstract class ResourceTestFixtures : IDisposable
         protected readonly Resources Resources;
         protected readonly IDispatcher Dispatcher;
         protected readonly World World;
-        
+        protected readonly string NewLineDelimiter = "\n";
```
Comment (tjaskula) [inline src/Vlingo.Http.Tests/Resource/ResourceTestFixtures.cs:40]: This is exactly the same as inlined `\n` before

File: src/Vlingo.Http/RequestParser.cs [hunk 2]
```diff
@@ -221,7 +223,7 @@ private string Compact()
             private Optional<string> NextLine(string errorResult, string errorMessage)
             {
                 var possibleCarriageReturnIndex = -1;
-                var lineBreak = _requestText.IndexOf("\n", _position, StringComparison.InvariantCultureIgnoreCase);
```
Comment (tjaskula) [inline src/Vlingo.Http/RequestParser.cs:224]: The invariant culture has to be there. It should not depend on the specific culture.

File: src/Vlingo.Http/RequestParser.cs [hunk 3]
```diff
@@ -221,7 +223,7 @@ private string Compact()
...
+                var lineBreak = _requestText.IndexOf(NewLineDelimiter, _position);
```
Comment (tjaskula) [inline src/Vlingo.Http/RequestParser.cs:226]: `IndexOf` supports both, string and char so maybe the difference is on your local culture and the fact that `StringComparison.InvariantCultureIgnoreCase` was used.

**Review by tjaskula — CHANGES_REQUESTED** [review]
Thanks for the contribution.
My gut feeling is that it's not dependent on Mac but maybe on your local culture.

Could you provide a Unit Test that proves the not working behavior was fixed ?

File: src/Vlingo.Http/RequestParser.cs [hunk 4]
```diff
@@ -32,6 +32,8 @@ public static RequestParser ParserFor(byte[] requestContent)
 
         public void ParseNext(byte[] requestContent) => _virtualStateParser.Includes(requestContent).Parse();
 
+        public static readonly char NewLineDelimiter = '\n';
```
Comment (hellxcz) [inline src/Vlingo.Http/RequestParser.cs:35]: here it is of type char.

**Comment by tjaskula** [comment]
Thanks @hellxcz for valuable feedback. I just addressed it in https://github.com/vlingo-net/vlingo-net-http/compare/380816e16a69...7b2d9c6d7fe6.

**Pull request closed without merging**